projects/demo/src/pages/documentation/what-is-maskito/what-is-maskito.component.ts

```diff
@@ -1,20 +1,15 @@
 import {ChangeDetectionStrategy, Component} from '@angular/core';
 import {RouterLink} from '@angular/router';
 import {DemoPath} from '@demo/constants';
-import {TuiAddonDocModule} from '@taiga-ui/addon-doc';
-import {TuiLinkModule} from '@taiga-ui/core';
-import {TuiIslandModule, TuiMarkerIconModule} from '@taiga-ui/kit';
+import {TuiAddonDoc} from '@taiga-ui/addon-doc';
+import {TuiLink} from '@taiga-ui/core';
+import {TuiAvatar} from '@taiga-ui/kit';
+import {TuiIslandDirective} from '@taiga-ui/legacy';
```

Comment:
Maybe switch to card?

Reply:
Done in https://github.com/taiga-family/maskito/pull/1390/commits/55a7b0932b92b96e5ef3434e94acb6f8a0f1defe